File: ITCR.InclusionesTec/ITCR.CommonClasses/Grupo.cs

#region Acerca de...
/////////////////////////////////////////////////////////////////////////////
//  Instituto Tecnológico de Costa Rica
//  Proyecto: Inclutec - Sistema Automatizado de Inclusiones
//  Descripción: Clase entidad para la tabla SIFGrupo
//  Generado por ITCR Gen v1.0.0.0 
//  Fecha: Domingo, 12 de Diciembre de 2013, 10:50:00 a.m.
////////////////////////////////////////////////////////////////////////////
#endregion

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ITCR.Ado.ClasesComunes
{
    public class Grupo
    {
        #region Propiedades

        /// <summary>
        /// Corresponde al identificador del grupo
        /// </summary>
        public int Id_Grupo { get; set; }

        /// <summary>
        /// Corresponde al numero del grupo
        /// </summary>
        public int Num_Grupo { get; set; }

        /// <summary>
        /// Corresponde al numero de cupos del grupo
        /// </summary>
        public int Num_Cupos { get; set; }

        /// <summary>
        /// Corresponde al numero de cupos extra del grupo
        /// </summary>
        public int Num_Cupos_Extra { get; set; }

        /// <summary>
        /// Corresponde al identificador del curso al que pertenece
        /// </summary>
        public int Id_Curso { get; set; }

        /// <summary>
        /// Corresponde a los horario del grupo
        /// </summary>
        public LinkedList<Horario> Li_Horarios { get; set; }

        #endregion
    }
}

File: ITCR.Inclutec/src/com/itcr/inclutec/clases_comunes/Grupo.java
package com.itcr.inclutec.clases_comunes;

import java.util.LinkedList;

public class Grupo {
	public int Id_Grupo, Num_Grupo, Num_Cupos, Num_Cupos_Extra, Id_Curso;
    public int getId_Grupo() {
		return Id_Grupo;
	}
	public void setId_Grupo(int id_Grupo) {
		Id_Grupo = id_Grupo;
	}
	public int getNum_Grupo() {
		return Num_Grupo;
	}
	public void setNum_Grupo(int num_Grupo) {
		Num_Grupo = num_Grupo;
	}
	public int getNum_Cupos() {
		return Num_Cupos;
	}
	public void setNum_Cupos(int num_Cupos) {
		Num_Cupos = num_Cupos;
	}
	public int getNum_Cupos_Extra() {
		return Num_Cupos_Extra;
	}
	public void setNum_Cupos_Extra(int num_Cupos_Extra) {
		Num_Cupos_Extra = num_Cupos_Extra;
	}
	public int getId_Curso() {
		return Id_Curso;
	}
	public void setId_Curso(int id_Curso) {
		Id_Curso = id_Curso;
	}
	public LinkedList<Horario> getLi_Horarios() {
		return Li_Horarios;
	}
	public void setLi_Horarios(LinkedList<Horario> li_Horarios) {
		Li_Horarios = li_Horarios;
	}
	public LinkedList<Horario> Li_Horarios;
}
